packages/uni_app/lib/view/home/widgets/exit_app_dialog.dart

```diff
@@ -36,7 +37,8 @@ class BackButtonExitWrapper extends StatelessWidget {
               ElevatedButton(
                 onPressed: () {
                   userActionCompleter.complete(true);
-                  Navigator.of(context).pop(false);
+                  SystemChannels.platform
+                      .invokeMethod<void>('SystemNavigator.pop');
```

Comment:
```suggestion
                  await SystemNavigator.pop()
```

Use SystemNavigator.pop instead of its implementation.
This way, we don't need to depend on potentially unstable platform channel methods

https://api.flutter.dev/flutter/services/SystemNavigator/pop.html

Reply:
You will need to make the closure async!